pages/platform/ai/deploy_guide_04_scaling_strategies/guide.en-gb.md

```diff
@@ -0,0 +1,114 @@
+
+## Requirements
+
+- a **Public cloud** project
+- access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
+- start deploying an app and get to **Step 3**: `Resources`
+
+## Scaling principles
+
+In the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), it is possible to select the **resources** in `Step 3` of the app deployment.
+
+This step allows you to choose between two scaling strategies: **static scaling** and **autoscaling**.
+
+### Static scaling
+
+The **static scaling** strategy allows you to choose the number of replicas on which the app will be deployed.
+
+The minimum number of replicas is **1** and the maximum is **10**.
+
+> [!warning]
+>
+> It is recommended to deploy on a **minimum of 2 replicas** to have high availability!
+>
+
+**When using static scaling?**
+
+- Static scaling can be used if you want to have fixed costs.
+
+- This scaling strategy is also useful when your consumption or inference load is fixed.
+
+### Autoscaling
+
+With the autoscaling strategy, you can play on several parameters.
+
+#### Minimum and maximum number of replicas
+
+With the autoscaling strategy, it is possible to choose both the **minimum number of replicas** (1 by default) and the **maximum number of replicas**.
+
+#### Monitored metric
+
+It is also possible to choose the metric to be monitored. This will act as a trigger for **autoscaling**. There are two metrics to choose from: `CPU` or `RAM`.
+
+#### Trigger threshold
+
+The threshold for the percentage of average use can also be chosen. It is an integer between 1 and 100%.
+
+The threshold of the average usage percentage that will trigger the scaling (up or down) of the app replicas.
+
+> [!primary]
+>
+> **High availability** will measure the average resource usage across its replicas and add instances if this average exceeds the specified average usage percentage threshold.
+>
+> Conversely, it will remove instances when this average resource utilisation falls below the threshold.
+>
+
+**When using autoscaling?**
+
+- You can use autoscaling when you have irregular or sawtooth inference loads.
+
+## Scaling example
+
+We will use the following example:  
+
+In case an app is based on the `AI1-1-CPU` flavor with a resource size of 2 (i.e. **2 CPUs**), this means that each replica of the application will be entitled to **2 vCores** and **8GiB RAM**.
+
+### Example 1
+
+First, we choose an `Autoscaling`.
+
+Then, we set the trigger threshold to `75%` of **CPU**.
+
+In this case, the app will be scaled when the average CPU usage across all its replicas is above **> 1.5 CPU (2*0.75)**, and it will be scaled when the average CPU usage falls below **< 1.5**.
```

Comment:
```suggestion
In this case, the app will be scaled up when the average CPU usage across all its replicas is above **> 1.5 CPU (2*0.75)**, and it will be scaled down when the average CPU usage falls below **< 1.5**.
```